src/cmd/gen-drone.ts

```diff
@@ -0,0 +1,83 @@
+import { readFileSync, writeFileSync } from 'fs'
+import { Argv } from 'yargs'
+import { OtomiDebugger, terminal } from '../common/debug'
+import { hfValues } from '../common/hf'
+import { BasicArguments, ENV } from '../common/no-deps'
+import { cleanupHandler, otomi, PrepareEnvironmentOptions } from '../common/setup'
+
+export interface Arguments extends BasicArguments {
+  dryRun: boolean
+  d: boolean
+  'dry-run': boolean
```

Comment:
Could we please have just one option, either one of these three, and the rest are aliases? I expect it is not necessary to set 3 options that have the same meaning?

Reply:
Sadly, no.

Yargs parses `--dry-run` and `-d` into these three separate objects. You cannot expect one or the other to change, without changing the other two.

Unless you know of a way to propagate the changes from one to the other 2?

Reply:
I don't understand what you mean... There is an alias option that will parse --dry-run and -d automatically right, so you only need one argument for the interface/type.

E.g. suppose I have this interface & options object:

```
interface Arguments extends BasicArguments {
  wait: string
}

const kindOptions: { [key: string]: Options } = {
  wait: {
    alias: 'w',
    describe: 'Wait for the cluster to be ready before running the test suite.',
    default: '0s',
    type: 'string',
  },
}

Then it doesn't seem necessary to set the aliases as a type...

Reply:
I see. But still what happens if you try to acces `args.w` instead of `args.wait` Since both are parsed, thus accessible (but not defined in the interface), the values are not linked. Could pose a problem, no?

Reply:
I think it would be very confusing to start using aliases for accessing variables tbh... Furthermore, I would expect a type error if you access a variable that is not defined in the Arguments interface.